api/Models/HotelDto.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace iTechArt.Hotels.Api.Models
```

Comment:
Please do not place database entities in the same folder with presentation layer models.
Also I would suggest using Entity suffix for all DB classes (e.g. HotelEntity) and calling presentation layer model just Hotel.